student/src/test/java/tds/student/services/remote/RemoteOpportunityServiceTest.java

```diff
@@ -283,4 +286,88 @@ public void shouldReturnFalseForValidationErrorReturned() throws ReturnStatusExc
     assertThat(isApproved).isFalse();
     verify(examRepository).updateStatus(oppInstance.getExamId(), statusChange.getStatus().name(), deniedStatus);
   }
-}
\ No newline at end of file
+  
+  @Test(expected = ReturnStatusException.class)
+  public void shouldThrowForErrorPresent() throws ReturnStatusException {
+    service = new RemoteOpportunityService(legacyOpportunityService, true, false, examRepository);
+    OpportunityInstance oppInstance = new OpportunityInstance(UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID());
+    final String assessmentKey = "assessmentKey";
+    Response<ExamConfiguration> errorResponse = new Response<>(new ValidationError("uh", "oh!"));
+  
+    when(examRepository.startExam(oppInstance.getExamId())).thenReturn(errorResponse);
+    service.startTest(oppInstance, assessmentKey, null);
+  }
+  
+  @Test(expected = ReturnStatusException.class)
+  public void shouldThrowForReturnStatusNotStarted() throws ReturnStatusException {
+    service = new RemoteOpportunityService(legacyOpportunityService, true, false, examRepository);
+    OpportunityInstance oppInstance = new OpportunityInstance(UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID());
+    final String assessmentKey = "assessmentKey";
+  
+    ExamConfiguration mockExamConfig = new ExamConfiguration.Builder()
```

Comment:
Not a huge deal to me, but I believe you could simply do `new ExamConfiguration.Builder().withStatus(ExamStatusCode.STATUS_DENIED).build()` and the test would still pass.

Unrelated to this change but I'm surprised the status is fetched from the configuration rather than the exam.